packages/coinstac-ui/app/render/state/ducks/auth.js

```diff
@@ -24,13 +24,13 @@ export const logout = applyAsyncLoading(() => {
   };
 });
 
-export const signUp = applyAsyncLoading(user => {
-  return (dispatch) => { // eslint-disable-line
-    return app.core.initialize(user)
-    .then(() => app.notify('success', 'New user created successfully'))
```

Comment:
Moved notification calls to the component. This makes things clearer: folks look at the component to see the control flow, everything in _app/render/state/_ should deal with **state only**.